robottelo/constants.py

```diff
@@ -21,6 +21,113 @@
     'CLOSED'
 ]
 
+CUST_PERMS_LIST = [
```

Comment:
Have to check whether adding changes to existing architecture , won't effect the other test cases , also the helper functions for permissions have to be updated to support new settings and if there are dependent test cases , they needs update as well.
So, it would be good to track it separately, creating a issue for it.